gulpfile.js

```diff
@@ -127,6 +127,16 @@ gulp.task('copy', ['less'], function() {
 });
 
 /**
+ * Deploy dist to static
+ */
+gulp.task('deploy', ['copy', 'dist'], function() {
```

Comment:
From what I understand these are all asynchronous requirements anyway.  `dist` never had a requirement for `copy.